lib/class-gb-rest-autosaves-controller.php

```diff
@@ -0,0 +1,355 @@
+<?php
+/**
+ * REST API: GB_REST_Autosaves_Controller class.
+ *
+ * @package WordPress
+ * @subpackage REST_API
+ * @since 5.0.0
+ */
+
+/**
+ * Core class used to access autosaves via the REST API.
+ *
+ * @since 5.0.0
+ *
+ * @see WP_REST_Controller
+ */
+class GB_REST_Autosaves_Controller extends WP_REST_Revisions_Controller {
```

Comment:
It's quite strange that a class _extends_ another but then _contains_ a instance of the latter. I get that it works; it's just conceptually a little harder to reason about.

Reply:
@danielbachhuber pointed this out earlier asking why we didn't just use the parent instead of instantiating the revisions controller. I'm open to changing this but found initialization was required for reusing the way I was trying to.